frontend/ui/widget/menu.lua

```diff
@@ -1131,21 +1123,24 @@ function Menu:mergeTitleBarIntoLayout()
         -- Title bar items can be accessed through key mappings on kindle
         return
     end
+    if Device:hasDPad() then
+        -- We'll mangle layout, drop the current focus while FocusManager can still find it
+        self:handleEvent(Event:new("Unfocus"))
+    end
     local menu_item_layout_start_row = 1
-    local titlebars = {self.title_bar, self.outer_title_bar}
-    for _, v in ipairs(titlebars) do
-        -- Menu uses the right key to trigger the context menu: we can't use it to move focus in horizontal directions.
-        -- So, add title bar buttons to FocusManager's layout in a vertical-only layout
-        local title_bar_layout = v:generateVerticalLayout()
-        for _, row in ipairs(title_bar_layout) do
-            table.insert(self.layout, menu_item_layout_start_row, row)
-            menu_item_layout_start_row = menu_item_layout_start_row + 1
-        end
+    -- Menu uses the right key to trigger the context menu: we can't use it to move focus in horizontal directions.
```

Comment:
For reference, this is no longer the case, unless you have hasFewKeys, all other devices have full dPad navigation functionality.

Reply:
I took that to mean that Menu (the class) binds Menu (the key), so it cannot uses FocusManager's own Menu bind.

Which is true (on SDL, because FocusManager binds Contextmenu, which we discussed a few hours ago.. somewhere ;p).

Reply:
Oh, it's talking about the `Right` key, lol.

I clearly need more sleep.